### ISSUE TITLE
Frozen the encoder and global point branch.

### ISSUE THREAD

**Issue by Wangweizhen-S** [yyfz/Pi3#139]
Dear Author, I have some questions regarding the training code and I hope you can provide answers. First of all, I noticed that in the paper it is mentioned that during the training process the entire encoder is frozen. Then, why was a learning rate set? Is this merely about training the geometric prediction head for the entire pi3 project? Additionally, I noticed that there is a global point cloud in the code. I observed that it uses the first frame for cross-attention. This seems to be different from the approach of the pi3. Is it necessary to enable this during training? Finally, if I want to make full-scale fine-tuning of pi3 or pi3x, should I enable the encoder and conduct the training in one go without going through the phased training process? (With the confidence level indicator included) Thank you for your response.

**Comment by yyfz** [comment]
Hi, [Wangweizhen-S](https://github.com/Wangweizhen-S)!

There was no learning rate set for the encoder; that was a typo, and I’ve fixed it in the latest arXiv version.

Regarding the global point head: if you are training from scratch (using DINOv2 for initialization instead of VGGT), using this will stabilize the training process.

If you are fine-tuning with limited data, please keep the encoder frozen (don’t "open" it). With more data, you can fine-tune all weights. For example, PI3X was developed by fine-tuning all the weights of PI3.

**Comment by Wangweizhen-S** [comment]
Thanks for your answer. It helps a lot
